uefi/flashregionsection.go

```diff
@@ -10,29 +10,44 @@ import (
 // FlashRegionSectionSize is the size of the Region descriptor. It is made up by 18 fields, each 16-bits large.
 const FlashRegionSectionSize = 36
 
+// Region contains a the start and end of a region in flash. This can be a BIOS, ME, PDR or GBE region.
+type Region struct {
+	Base  uint16
+	Limit uint16
+}
+
+// Available checks to see if a region is valid (has nonzero limit)
+func (r *Region) Available() bool {
+	return r.Limit > 0
+}
+
+func (r *Region) String() string {
```

Comment:
me, personally, I would prefer hex but.

Reply:
done. This isn't actually used right now, json exports as decimal anyway.